### PR TITLE
[Docs] Docupdate

### PR DESCRIPTION
**Pull request by ianegordon**
Documentation clarifications and updates from Build Cop rotation.

### CODE REVIEW

File: contributing/releasing.md [hunk 1]
```diff
@@ -87,13 +92,17 @@ you will need to examine the release's changes.
...
+Use scripts/release/diff to determine API changes
 
-Generate the API diff by running the following:
+Inspect changes to public component headers and manually generate the API diff by hand.
+
+    scripts/release/diff components/*/src/*.h
+
+~~Generate the API diff using scripts/release/api_diff:~~
+
+**The api_diff script is broken. Manually generate the API diff using scripts/release/diff as noted above and continue to the next step, Commit the changes.**
```
Comment (willlarche) [inline contributing/releasing.md]: Mark with date and GH username.

File: contributing/releasing.md [hunk 2]
```diff
@@ -182,14 +191,19 @@ Commit the results to your branch:
     git commit -am "Bumped version number to $(pod ipc spec MaterialComponents.podspec | grep '"version"' | cut -d'"' -f4)."
     git push origin release-candidate
 
-#### Lint the local podspec
+#### Verify Cocoapods
```
Comment (willlarche) [inline contributing/releasing.md]: Verify Cocoapods _[what]_

Comment (willlarche) [reply contributing/releasing.md]: Also, the 'p' should be uppercase.

File: contributing/releasing.md [hunk 3]
```diff
@@ -182,14 +191,19 @@ Commit the results to your branch:
...
+
+Send our local podspec through the Cocoapods linter:
```
Comment (willlarche) [inline contributing/releasing.md]: uppercase the 'p'

File: contributing/releasing.md [hunk 4]
```diff
@@ -182,14 +191,19 @@ Commit the results to your branch:
...
 
     pod lib lint MaterialComponents.podspec
 
-Note: Ensure that you can [push the podspec](#publish-to-cocoapods) later by checking for `MaterialComponents` in your `Pods` when you:
+Cocoapods publishes a directory of publicly available pods through its **trunk** service.
```
Comment (willlarche) [inline contributing/releasing.md]: the 'p'

**Review by willlarche — CHANGES_REQUESTED** [review]

File: contributing/releasing.md (hunk 1 above)
Comment (ianegordon) [reply contributing/releasing.md]: Done.

File: contributing/releasing.md (hunk 2 above)
Comment (ianegordon) [reply contributing/releasing.md]: Done.

File: contributing/releasing.md (hunk 4 above)
Comment (ianegordon) [reply contributing/releasing.md]: Done.

**Comment by ianegordon** [comment]
Many Thanks!
PTAL.

**Review by willlarche — APPROVED** [review]

**Pull request merged**